util/retry.go

```diff
@@ -29,12 +25,10 @@ func Retry(rfn RetryFn, times int, interval time.Duration) error {
 		if times == 0 {
 			return err
 		}
-		if awsErr, ok := err.(awserr.Error); ok {
-			if awsErr.Code() == AWSThrottlingCode {
-				log.Get().Log("func", "utils.Retry", "msg", "waiting for Throttling error", "times-left", times)
-				time.Sleep(interval)
-				return Retry(rfn, times, interval)
-			}
+		if request.IsErrorRetryable(err) || request.IsErrorThrottle(err) || request.IsErrorExpiredCreds(err) {
```

Comment:
If we retry on an expired credential, it will fail again no?

Reply:
I mean, if you had an invalid credential in the first place? You shouldn't have reached here.
And if it expired, I guess the `Retryable` will be set to false at some point.

Reply:
If the credentials are wrong it'll fail on the initialization, thats basically it. This is what AWS uses for checking if something has to be retries so I just did the same.